cinema/serializers.py

```diff
@@ -22,3 +55,23 @@
         instance.save()
 
         return instance
+
+
+class CinemaHallSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for `CinemaHallSerializer` to leverage automatic field handling and reduce boilerplate code.